docs/apache-hive-catalog.md

```diff
@@ -42,6 +42,9 @@ The Hive catalog supports creating, updating, and deleting databases and tables
 
 When you use the Gravitino with Trino. You can pass the Trino Hive connector configuration using prefix `trino.bypass.`. For example, using `trino.bypass.hive.config.resources` to pass the `hive.config.resources` to the Gravitino Hive catalog in Trino runtime.
 
+When you use the Gravitino with Spark. You can pass the Spark Hive connector configuration using prefix `spark.bypass.`. For example, using `spark.bypass.hive.config.resources` to pass the `hive.config.resources` to the Spark Hive connector in Spark runtime.
```

Comment:
Are you sure these two configs `spark.bypass.hive.config.resources` and `hive.config.resources` works in Spark?

Reply:
`hive.config.resources` is just a config example, change it to a real config like `hive.exec.dynamic.partition.mode`?

Reply:
Why do you use a non-existent config as an example, this is really confusing. Please think of as a user who reads your document, does he really understand what you want to describe?

Reply:
got it, fixed